content/zh/docs/glossary/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "术语表" 
+description: "用于描述Helm体系结构的组件。"
+weight: 9
+---
+
+# 术语表
+
+## Chart
+
+Helm包涵盖了将Kubernetes资源安装到Kubernetes集群所需要的足够多的信息。
+
+Charts包含`Chart.yaml`文件和模板，默认值(`values.yaml`)，以及相关依赖。
+
+Charts开发设计了良好定义的目录结构，并且打包成了一种称为 _chart archive_ 文件格式。
+
+## Chart包
+
+Chart包(_chart archive_)是被tar和gzip压缩(并且可选签名)的chart.
+
+## Chart依赖 (子chart)
+
+Chart可以依赖于其他的chart。 依赖可能会以以下两种方式出现:
+
+- 软依赖： 如果另一个chart没有在集群中安装，chart可能会无法使用。Helm未对这个案例提供工具。
+这个案例中，依赖会被分别管理。
+- 硬依赖： 一个chart可以包含 (在它的`charts/`目录中) 另一个它所依赖的chart。这个案例中，
+安装chart的同时会安装所有依赖。chart和它的依赖会作为一个集合进行管理。
+
+当一个chart(通过`helm package`)打包时所有的依赖都会和它绑定。
```

Comment:
Does this clarify that only "hard" dependencies and not "soft" ones will be packaged with the chart?

Reply:
Yes, it's different from soft dependency.